### PR TITLE
refactor:  replace the keystore command with account cmd

### PR DESCRIPTION
**Pull request by flywukong**
### Description
1. remove default password file loading logic for safe
2. support  "account new " , "account export " and "account import" command
3.  replace the keystore command with account cmd
4. support version command

### Rationale

improve usage experiment

### Example
```
qa-test % ./gnfd-cmd account new
Please enter a passphrase now:
create new account: {0x21c1228786b93C656206Ccc6a753E0831B81023f} successfully
qa-test % ./gnfd-cmd account list
Account : { 21C1228786B93C656206CCC6A753E0831B81023F },  keystore /Users/user/.gnfd-cmd/keystore/key.json


qa-test % ./gnfd-cmd version
Greenfield Cmd Version: v0.0.9-alpha.1
```

### Changes

Notable changes:
* support account cmd

### CODE REVIEW

File: cmd/cmd_keystore.go [hunk 1]
```diff
@@ -6,67 +6,73 @@ import (
...
+		Usage:     "import the account by the private key file",
 		ArgsUsage: "[ <keyfile> ] ",
 		Description: `
-generate a keystore file to manage user's private key information.
+Import account info from private key file and generate a keystore file to manage user's private key information.
+If no keyfile is specified, a keystore will be generated at the default path （homedir/.gnfd-cmd/keystore/key.json）
+
 Examples:
-$ gnfd-cmd keystore generate --privKeyFile key.txt  `,
+$ gnfd-cmd  account import --privKeyFile  key.txt  ./key.json`,
```
[the remark this answers is not in the document]
Comment (flywukong) [reply cmd/cmd_keystore.go]: yes, improved it.
```
qa-test % ./gnfd-cmd  -k key.json  account import  key.txt3
Please enter a passphrase now:
import account successfully, key address: 0x54a22B39A51E3446b70A27F1d68753267EBa5745, encrypted key file: key.json
```

File: cmd/cmd_account.go [hunk 2]
```diff
@@ -2,243 +2,92 @@ package main
...
-	if err != nil {
-		return toCmdErr(err)
-	}
-	c, createPaymentAccount := context.WithCancel(globalContext)
-	defer createPaymentAccount()
-	acc, err := client.GetDefaultAccount()
-	if err != nil {
-		return toCmdErr(err)
+// key.txt contains the origin private hex string 
+$ gnfd-cmd  -k key.json  account import  key.txt `,
```
Comment (KeefeL) [inline cmd/cmd_account.go:32]: move -k key.json behind the import key.txt is better?

**Review by KeefeL — APPROVED** [review]

File: cmd/cmd_account.go (hunk 2 above)
Comment (flywukong) [reply cmd/cmd_account.go:32]: This is a global flag, which does not belong to the account command. All other commands share this flag, so needs to put it in front.

Comment (flywukong) [reply cmd/cmd_account.go:32]: If put it in the back,  we need to add a new flag which just belong to the "account import "command

**Review by KeefeL — APPROVED** [review]

**Pull request merged**